flake.nix

```diff
@@ -1,48 +1,76 @@
 {
   description = "devshell";
   # To update all inputs:
-  # $ nix flake update --recreate-lock-file
-  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
-  inputs.systems.url = "github:nix-systems/default";
-
-  outputs = { self, nixpkgs, systems }:
-    let
-      eachSystem = nixpkgs.lib.genAttrs (import systems);
-    in
-    {
-      legacyPackages = eachSystem (system:
-          import self {
-            inherit system;
-            inputs = null;
-            nixpkgs = nixpkgs.legacyPackages.${system};
-          }
-      );
-
-      templates = rec {
-        toml = {
-          path = ./templates/toml;
-          description = "nix flake new my-project -t github:numtide/devshell";
-        };
-        flake-parts = {
-          path = ./templates/flake-parts;
-          description = "nix flake new my-project -t github:numtide/devshell#flake-parts";
+  # nix flake update --recreate-lock-file
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/efc960b6d6a6498c23868f4ba59fcb8bb51c9861";
```

Comment:
Can you point to a channel instead, to make it easier to bump nixpkgs?

Reply:
Done in https://github.com/numtide/devshell/pull/289/commits/bd708b4c66bd1d00459136340c8f90330b0acb6e